base/abstractarray.jl

```diff
@@ -1835,8 +1835,8 @@ end
 
 ## N argument
 
-ith_all(i, ::Tuple{}) = ()
-ith_all(i, as) = (as[1][i], ith_all(i, tail(as))...)
+@inline ith_all(i, ::Tuple{}) = ()
+@inline ith_all(i, as) = (as[1][i], ith_all(i, tail(as))...)
```

Comment:
I was thinking of non-`map_n!` consumers of `ith_all`. But grep'ing suggests there are no such consumers in base, so please ignore this suggestion :).